nuke-from-orbit/loadtester

```diff
@@ -0,0 +1,372 @@
+  aws_session_token=$(cat $config_file | jq -r '.aws_session_token // empty')
+  lookml_project_repo=$(cat $config_file | jq -r '.lookml_project_repo // empty')
+
+  if [ -z $gcp_oauth_client_id ]; then echo "Could not find gcp_oauth_client_id"; exit 1; fi
+  if [ -z $gcp_oauth_client_secret ]; then echo "Could not find gcp_oauth_client_secret"; exit 1; fi
+  if [ -z $gcp_iap_email ]; then echo "Could not find gcp_iap_email"; exit 1; fi
+
+  if ! [ $setup_command == 'self-contained' ]
+  then
+    if [ -z $gcp_project_id ]; then echo "Could not find gcp_project_id"; exit 1; fi
+    if [ -z $loadtest_name ]; then echo "Could not find loadtest_name"; exit 1; fi
+    if [ -z $loadtest_dns_domain ]; then echo "Could not find loadtest_dns_domain"; exit 1; fi
+    if [ -z $gcp_zone ]; then echo "Could not find gcp_zone"; exit 1; fi
+    if [ -z $gcp_cluster_node_count ]; then echo "Could not find gcp_cluster_node_count"; exit 1; fi
+    if [ -z $gcp_cluster_machine_type ]; then echo "Could not find gcp_cluster_machine_type"; exit 1; fi
+    if [ -z $looker_user ]; then echo "Could not find looker_user"; exit 1; fi
+    if [ -z $looker_pass ]; then echo "Could not find looker_pass"; exit 1; fi
+  else
+    if [ -z $aws_access_key ]; then echo "Could not find aws_access_key"; exit 1; fi
+    if [ -z $aws_secret_key ]; then echo "Could not find aws_secret_key"; exit 1; fi
+    if [ -z $aws_session_token ]; then echo "Could not find aws_session_token. It may not be required, but be aware."; fi
+    if [ -z $lookml_project_repo ]; then echo "Could not find lookml_project_repo"; exit 1; fi
+    if [ -z $gcp_iap_email ]; then echo "Could not find gcp_iap_email"; exit 1; fi
+  fi
+
+}
+
+# Build (or rebuild) the load test image
+build_loadtest_image() {
+  gcloud builds submit --tag gcr.io/${gcp_project_id}/${loadtest_name}:${image_tag} docker-image/.
+}
+
+# Configure access to kubernetes cluster via gcloud
+get_kubernetes_creds() {
+  gcloud container clusters get-credentials $loadtest_name \
+    --zone $gcp_zone \
+    --project $gcp_project_id
+}
+
+# Set the oauth secret (used by iap)
+set_oauth_secret() {
+  kubectl create secret generic iap-secret \
+    --from-literal=client_id=$gcp_oauth_client_id \
+    --from-literal=client_secret=$gcp_oauth_client_secret
+}
+
+# Set the Looker username and password secret
+set_looker_secret() {
+  kubectl create secret generic website-creds \
+    --from-literal=username=$looker_user \
+    --from-literal=password=$looker_pass
+}
+
+# Self-contained only: Set AWS credentials for monitoring
+set_aws_secret() {
+  kubectl create secret generic aws-creds \
+    --from-literal=aws-access-key=$aws_access_key \
+    --from-literal=aws-secret-key=$aws_secret_key \
+    --from-literal=aws-session-token=$aws_session_token
+}
+
+# Deploy GCP managed SSL certificate
+deploy_managed_certificate() {
+  kubectl apply -f ${config_dir}/loadtest-cert.yaml
```

Comment:
Maybe find this team and see if there's a buganizer for how slow this is, if not, you could open a ticket